hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/source/stats/TestExpressionEvaluator.java

```diff
@@ -77,291 +88,291 @@ public class TestExpressionEvaluator {
     ValueLiteralExpression vExpr = new ValueLiteralExpression(12);
 
     RowData indexRow1 = intIndexRow(11, 13);
-    greaterThanOrEqual.bindFieldReference(rExpr)
-        .bindVal(vExpr)
-        .bindColStats(indexRow1, queryFields(2), rExpr);
-    assertTrue(greaterThanOrEqual.eval(), "12 < 13");
+    greaterThanOrEqual.bindVal(vExpr)
+        .bindFieldReference(rExpr);
+    Map<String, ColumnStats> stats1 = convertColumnStats(indexRow1, queryFields(2));
+    assertTrue(greaterThanOrEqual.eval(stats1), "12 < 13");
 
     RowData indexRow2 = intIndexRow(12, 13);
-    greaterThanOrEqual.bindColStats(indexRow2, queryFields(2), rExpr);
-    assertTrue(greaterThanOrEqual.eval(), "min 12 = 12");
+    Map<String, ColumnStats> stats2 = convertColumnStats(indexRow2, queryFields(2));
+    assertTrue(greaterThanOrEqual.eval(stats2), "min 12 = 12");
 
     RowData indexRow3 = intIndexRow(11, 12);
-    greaterThanOrEqual.bindColStats(indexRow3, queryFields(2), rExpr);
-    assertTrue(greaterThanOrEqual.eval(), "max 12 = 12");
+    Map<String, ColumnStats> stats3 = convertColumnStats(indexRow3, queryFields(2));
+    assertTrue(greaterThanOrEqual.eval(stats3), "max 12 = 12");
 
     RowData indexRow4 = intIndexRow(10, 11);
-    greaterThanOrEqual.bindColStats(indexRow4, queryFields(2), rExpr);
-    assertFalse(greaterThanOrEqual.eval(), "max 11 < 12");
+    Map<String, ColumnStats> stats4 = convertColumnStats(indexRow4, queryFields(2));
+    assertFalse(greaterThanOrEqual.eval(stats4), "max 11 < 12");
 
     RowData indexRow5 = intIndexRow(13, 14);
-    greaterThanOrEqual.bindColStats(indexRow5, queryFields(2), rExpr);
-    assertTrue(greaterThanOrEqual.eval(), "12 < 13");
+    Map<String, ColumnStats> stats5 = convertColumnStats(indexRow5, queryFields(2));
+    assertTrue(greaterThanOrEqual.eval(stats5), "12 < 13");
 
     RowData indexRow6 = intIndexRow(null, null);
-    greaterThanOrEqual.bindColStats(indexRow6, queryFields(2), rExpr);
-    assertFalse(greaterThanOrEqual.eval(), "12 <> null");
+    Map<String, ColumnStats> stats6 = convertColumnStats(indexRow6, queryFields(2));
+    assertFalse(greaterThanOrEqual.eval(stats6), "12 <> null");
 
     greaterThanOrEqual.bindVal(new ValueLiteralExpression(null, DataTypes.INT()));
-    assertFalse(greaterThanOrEqual.eval(), "It is not possible to test for NULL values with '>=' operator");
+    assertFalse(greaterThanOrEqual.eval(stats1), "It is not possible to test for NULL values with '>=' operator");
   }
 
   @Test
   void testIn() {
-    ExpressionEvaluator.In in = ExpressionEvaluator.In.getInstance();
+    In in = In.getInstance();
     FieldReferenceExpression rExpr = new FieldReferenceExpression("f_int", DataTypes.INT(), 2, 2);
 
     RowData indexRow1 = intIndexRow(11, 13);
-    in.bindFieldReference(rExpr)
-        .bindColStats(indexRow1, queryFields(2), rExpr);
-    in.bindVals(12);
-    assertTrue(in.eval(), "11 < 12 < 13");
+    in.bindFieldReference(rExpr);
+    in.bindVals(11, 12);
```

Comment:
In this pr, I fix a little bug in `In` evaluator.
Modify test case which IN expression has multiple literals to validate the behavior is correct now.